routes/routes.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
```

Comment:
Maličká dropbnosť, ale navrhoval by som sem pridať jeden malý doc-bloc (aspoň dočasne, nech sa IDE nezblázni)

```suggestion
/**
 * @var \App\Functions\Router $router - defined in @see public/index.php
 */
```